tests/code_layout/acceptable_missing_doc.py

```diff
@@ -1505,6 +1505,7 @@
     ],
     "QgsStackedDiagramProperties": [
         "apply()",
+        "rowsMoved()",
```

Comment:
Can you add documentation instead of expanding the missing doc list?

Reply:
Method removed